software/slink.yml

```
@@ -0,0 +1,10 @@
+name: "Slink"
+website_url: "https://docs.slinkapp.io/"
+source_code_url: "https://github.com/andrii-kryvoviaz/slink"
+description: "Image sharing platform built with Symfony and SvelteKit."
```

```suggestion
description: "Image sharing platform designed to give users complete control over their media sharing experience."
```

Better description, technical details/languages are not relevant in descriptions